BAZProject/MovieAPI.swift

```diff
@@ -18,18 +20,33 @@ class MovieAPI {
         else {
             return []
         }
-
-        var movies: [Movie] = []
-
-        for result in results {
-            if let id = result.object(forKey: "id") as? Int,
-               let title = result.object(forKey: "title") as? String,
-               let poster_path = result.object(forKey: "poster_path") as? String {
-                movies.append(Movie(id: id, title: title, poster_path: poster_path))
-            }
+        do {
+            let dataInfo = try? JSONSerialization.data(withJSONObject: results, options: .prettyPrinted)
+            return saveMoviesData(dataMovies: dataInfo!)
```

Comment:
Evitar hacer el force unwrap the dataInfo, mejor usar un `if let` o `guard let`.